en/simulation/multi_vehicle_jmavsim.md

```diff
@@ -22,21 +22,25 @@ To start multiple instances (on separate ports):
    ```
    ./Tools/sitl_multiple_run.sh 2
    ```
-1. Start the first instance:
+1. Start the first instance in the same terminal (this will run in the foreground):
    ```
    ./Tools/jmavsim_run.sh -l
    ```
-1. Start subsequent instances, specifying the *simulation* TCP port for the instance:
+1. Open a new terminal for each subsequent instance, specifying the *simulation* TCP port for the instance:
    ```
-   ./Tools/jmavsim_run.sh -p 4561 -l
+   ./Tools/jmavsim_run.sh -p 4560 -l
    ```
-   The port should be set to `4560+i` for `i` in `[0, N-1]`.
+   The port should be set to `4560+i` where `i` iterates for each instance (from `0` to `N-1`)
+ 
+*QGroundControl* should automatically connect to all the new vehicle instances (all GCS traffic is sent to PX4's remote UDP port: `14550` from all instances).
```

Comment:
I removed the numbering from here. The instructions/list are for "how to start the instances". That finished once the starting part is done. Now we go onto "how you test it". It doesn't have to be done this way, but it shouldn't be a continuation of the old list.